README.md

```diff
@@ -14,7 +14,7 @@ Some key features include:
 * tagging and tag-based content organization
 * email notification subscriptions for tags and comments
 * a barebones search interface
-* a user dashboard [presenting recent activity](https://publiclab.org/research)
+* an user dashboard [presenting recent activity](https://publiclab.org/research)
```

Comment:
hmm, i don't think this is correct, but it could be an American English convention?